concert/readers.py

```diff
@@ -114,7 +123,18 @@ def _get_num_images_in_file_real(self):
         return len(self._file.pages)
 
     def _read_real(self, index):
-        return self._file.pages[index].asarray()
+        image = self._file.pages[index].asarray().view(ImageWithMetadata)
+        if self._metadata_file:
+            image.metadata = self._json_metadata[str(index)]
+        else:
+            try:
+                image.metadata = ast.literal_eval(self._file.pages[index].description)
```

Comment:
`self._file.pages[index].description` doesn't return a dict? I am wondering why you need the `literal_eval`.

Reply:
No. It returns a string, that is formated like a dictionary assignment. This fits so well with my other tiff experiences...

Reply:
Or do you think I do something wrong when writing the metadata? The ImageDescription from the tests are shown like this in ImageJ "ImageDescription: {"time": "2022-10-14T12:25:37.739517", "random_value": 0.9344031366187244, "shape": [10, 10]}". And the shape is automatically added by tifffile. So I guess the writing is correct like this.